.clang-tidy

```diff
@@ -13,7 +13,15 @@ Checks: >
   modernize-type-traits,
   modernize-use-nullptr,
   modernize-use-using,
+  modernize-use-override,
+  modernize-use-equals-delete,
+  modernize-use-equals-default,
+  modernize-use-bool-literals,
+  modernize-loop-convert,
```

Comment:
Do not attempt to enable more checks as part of this PR.
It is unclear that we want to enable them and we would only treat them one by one.

Reply:
I've reverted these changes.